samples/net/openthread/coap_client_ble/Kconfig

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+menu "Zephyr Kernel"
+source "Kconfig.zephyr"
+endmenu
+
+module = COAP_CLIENT_BLE
+module-str = CoAP client with NUS
+source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+
+module = BLE_UTILS
```

Comment:
ble_utils has been prepared to act as separate module. It contains all necessary parts for handling BLE connections. At current form can be easily reused in other examples, so that I would stick the current implementation.